src/GitVersion.Configuration/ConfigurationFileLocator.cs

```diff
@@ -65,7 +74,7 @@ private void WarnAboutAmbiguousConfigFileSelection(string? workingDirectory, str
 
         if (!hasConfigInProjectRootDirectory && !hasConfigInWorkingDirectory)
         {
-            if (this.ConfigurationFile is not (DefaultFileName or DefaultAlternativeFileName))
+            if (!PossibleConfigFileNames.Any(entry => entry.Equals(this.ConfigurationFile)))
```

Comment:
Should we make the file comparison here case insensitive as well? I suppose that could be considered a breaking change, as someone might be crazy enough to have both `gitversion.yml` and `GitVersion.yml` on an ext4 or similar case sensitive file system, but I think `.gitversion.yml` somehow looks more Unix idiomatic than `.GitVersion.yml`. Thoughts?

Reply:
I agree, let's make it insensitive, I don't think we should treat it as breaking change as this does not affect most of the users.